### PR TITLE
TL/24 simple design patterns singleton page object

### PR DESCRIPTION
**Pull request by verginia**
Create project folder for Simple Design Patterns, Singleton, Page Object theme + README file

### CODE REVIEW

File: TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# Design Patterns
+***
+
+## Contents
+1. [Simple Design Patterns.](#simple-design-patterns)
```
Comment (tascaV) [inline TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md:5]: remove ~Simple~ or change to **About Design Patterns**

File: TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+2. [Singleton](#singleton)
+3. [Page Object](#page-object)
+
+
+## 1. Simple Design Patterns.
```
Comment (tascaV) [inline TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md:10]: remove ~Simple~ or change to **About Design Patterns**

**Review by tascaV — CHANGES_REQUESTED** [review]
Page Object Pattern is missing the page functionality description, beside locators we can have and page functionality defined by class methods etc,
If you could add a page object example in Java would be very nice. Thank you.

File: TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md (hunk 2 above)
Comment (verginia) [reply TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md]: done

File: TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md (hunk 1 above)
Comment (verginia) [reply TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md]: done

**Review by endalexei — APPROVED** [review]

**Review by tascaV — APPROVED** [review]

**Pull request merged**